packages/shared/src/features/onboarding/hooks/useInitFunnelPaddle.ts

```diff
@@ -1,36 +0,0 @@
-import { useSetAtom } from 'jotai/react';
```

Comment:
We can now just wrap the funnel stepper with PaymentContext for simplicity. All we need to track is when the checkout completes which the context can accept as a parameter.